src/react-components/room/useAvatarVolume.js

```diff
@@ -1,40 +1,31 @@
 import { useCallback, useEffect, useState } from "react";
 import { getPlayerInfo } from "../../utils/component-utils";
-import {
-  calcLevel,
-  calcLevelStepDown,
-  calcLevelStepUp,
-  calcGainMultiplier,
-  MAX_VOLUME_LABELS
-} from "../../utils/avatar-volume-utils";
 
-const minLevel = 0;
-const maxLevel = MAX_VOLUME_LABELS - 1;
-
-export default function useAvatarVolume(sessionId) {
+export default function useAvatarVolume(sessionId, onMultiplierChanged) {
   const playerInfo = getPlayerInfo(sessionId);
   const controlsEl = playerInfo?.el.querySelector("[avatar-volume-controls]");
   const controls = controlsEl?.components["avatar-volume-controls"];
-  const [levelStep, setLevelStep] = useState(0);
-  const [level, setLevel] = useState(0);
+  const [prevMultiplier, setPrevMultiplier] = useState(0);
```

Comment:
This is currently only being used to set the dynamic step size. If that actually turns out not to be a requirement this can go as well, further simplifying things.